Library/Homebrew/cmd/test-bot.rb

```diff
@@ -437,10 +438,35 @@ def formula(formula_name)
         formula_name
       end
 
-      test "brew", "uses", canonical_formula_name
-
       formula = Formulary.factory(canonical_formula_name)
 
+      if ARGV.include? "--fast"
```

Comment:
Can we put all of this in a standalone method like `fast_formula_test` or something and use it in [here](https://github.com/mikemcquaid/homebrew/blob/test-bot-master-fast/Library/Homebrew/cmd/test-bot.rb#L765-L767) like:
```ruby
if ARGV.include? "--fast"
  formulae.each { |f| fast_formula_test(f) }
else
  formulae.each { |f| formula(f) }
end
```

Personally, I found long method with early return on different function is hard to follow.